doc/md/canister-maintenance/compatibility.md

```diff
@@ -289,4 +333,30 @@ Stable interface compatibility check issued a WARNING for canister ...
  will be discarded. This may cause data loss. Are you sure?
 ```
 
+### Solution 2 using a migration function and single upgrade
+
+Instead of the previous two step solution, we can upgrade in one step using a migration function.
+
+1. Define a migration module and function that transform the old stable variable, at its current type, into the new stable variable at its new type.
+
+
+``` motoko no-repl file=../examples/CardMigration.mo
+```
+
+2. Specify the migration function as the migration expression of your actor declaration:
+
+
+``` motoko no-repl file=../examples/Card-v1c.mo
+```
+
+**After** we have successfully upgraded to this new version, we can also upgrade once more to a version that drops the migration code.
```

Comment:
Does this mean, the `.most` file will go back to version 1.0.0?

Reply:
Yes